ARCs/arc-0076.md

```diff
@@ -0,0 +1,64 @@
+---
+arc: 76
+title: H999 Account
+description: SHA512-256 999 account
+author: Ludovit Scholtz (@scholtz)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/208
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-06-12
+---
+
+## Abstract
+This standard specifies a computation for seed bytes for H999 account. For general adoption it is better for people to remember passphrase than mnemonic. With this standard person can hash the passphrase and receive the seed bytes for X25529 algorand account.
+
+## Motivation
+By providing a clear and precise computation process, H999 account empowers individuals to effortlessly obtain their seed bytes for algorand account. In the realm of practicality and widespread adoption, the standard highlights the immense advantages of utilizing a passphrase rather than a mnemonic. Remembering a passphrase becomes the key to unlocking a world of possibilities. With this groundbreaking standard, individuals can take control of their X25529 Algorand account by simply hashing their passphrase and effortlessly receiving the corresponding seed bytes. It's time to embrace this new era of accessibility and security, empowering yourself to reach new heights in the world of H999 accounts. Let this standard serve as your guiding light, motivating community to embark on a journey of limitless possibilities and unparalleled success.
+
+This standard seek the synchronization between wallets which may provide password protected accounts.
+
+## Specification
+
+Hash of the seed bytes is calculated with algorithm:
+
+```
+const init = `${data}`;
+let hash = sha512.sha512_256.hex(init);
+for (let i = 1; i <= 997; i++) {
+  hash = sha512.sha512_256.hex(hash);
+}
+const bytes = sha512.sha512_256.array(hash);
+
+const uint8 = new Uint8Array(bytes);
+const mnemonic = algosdk.mnemonicFromSeed(uint8);
+const genAccount = algosdk.mnemonicToSecretKey(mnemonic);
+```
+
+Length of the data section SHOULD be at least 16 bytes long.
+
+### Email Password account
+
+Email H999 account is account generated from the original data
+
+```
+let init = `${email}-${password}`;
```

Comment:
It is important to allow updates of the way things are derived, so one part of the passphrase needs to encode that.

Reply:
good idea